vizro-core/tests/unit/vizro/models/test_page.py

```diff
@@ -98,3 +99,8 @@ def test_action_auto_generation_valid(self, standard_px_chart):
 
 
 # TODO: Add unit tests for private methods in page build
+def test_page_build_left_side_removed(standard_px_chart):
+    page = vm.Page(title="Single Page", components=[vm.Graph(id="scatter_chart", figure=standard_px_chart)])
+    vm.Dashboard(pages=[page]).build()
+    assert ("className='left_side'" not in str(page.build())) is True
+    del dash.page_registry["Single Page"]
```

Comment:
Is there a way to clear the entire registry? In that case it might make sense to have a fixture for that, that we can attach to any test that needs it, instead of doing it manually at the end

Reply:
Unfortunately not! But this is what I initially tried e.g. `dash.page_registry = OrderedDict()` - however, this permanently clears the dash.page_registry and we end up having interdependence between the tests again.